glib/tests/mappedfile.c

```diff
@@ -86,6 +86,7 @@ test_writable (void)
   gchar *tmp_copy_path;
 
   tmp_copy_path = g_build_filename (g_get_user_runtime_dir (), "glib-test-4096-random-bytes", NULL);
+  g_mkdir_with_parents (g_get_user_runtime_dir (), 0700);
```

Comment:
I think this is upstream as well https://git.gnome.org/browse/glib/commit/?id=7c6141a546b80866e26a12cee8b6a9f1be5b9fc6